adagrams/game.py

```diff
@@ -1,11 +1,212 @@
+
+import random
+LETTER_POOL = {
```

Comment:
Great use of constant variables, especially outside of functions. These variables could live inside of functions but they definitely add more clutter to the function body than necessary.

Having constant variables or any other data containing large chunks allows the function body to focus on the important logic (aka the purpose of the function).